crowbar_framework/app/models/node_object.rb

```diff
@@ -170,6 +170,10 @@ def self.all
     self.find nil
   end
 
+  def self.admin_node
+    all.map { |n| n if n.admin? }.first
```

Comment:
well, this has to be a select, map is totally wrong here. i just found out by the test suite.